<launch>

    <!-- Robot State Publisher -->
    <param name="robot_description" command="$(find xacro)/xacro $(find charlie)/urdf/test.urdf.xacro" />

    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" output="screen" />

    <!-- RViz -->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find charlie)/rviz/urdf.rviz" required="true" />
    
    <!-- Joint State Publisher Script -->
    <node name="joint_state_publisher" pkg="charlie" type="jointstate_pub.py" output="screen" />

</launch>

